--- a/src/include/pg_config_manual.h
+++ b/src/include/pg_config_manual.h
@@ -67,7 +67,7 @@
  * Changing this requires an initdb.
  */
-#define INDEX_MAX_KEYS		32
+#define INDEX_MAX_KEYS   64

 /*
  * Maximum number of columns in a partition key
  */
@@ -201,7 +201,7 @@
  * support them yet.
  */
 #ifndef WIN32
-#define DEFAULT_PGSOCKET_DIR  "/tmp"
+#define DEFAULT_PGSOCKET_DIR  "/run/postgresql"
 #else
 #define DEFAULT_PGSOCKET_DIR ""
 #endif
